network-area-diagram/src/test/resources/IEEE_14_bus_fictitious.svg

```diff
@@ -679,7 +679,7 @@ path.nad-arrow-in:not(.nad-state-in .nad-arrow-in) {visibility: hidden}
     </g>
     <g class="nad-text-nodes">
         <foreignObject id="0-textnode" y="-937.76" x="479.50" height="1" width="1">
-            <div xmlns="http://www.w3.org/1999/xhtml" class="nad-label-box">
+            <div xmlns="http://www.w3.org/1999/xhtml" class="nad-label-box" style="position: absolute; top: 0px; left: 0px" id="0-textnode">
```

Comment:
could we avoid to repeat the name space in all the divs or is it required? Or is it allowed to put it in the foreignObject?